.editorconfig

```diff
@@ -21,6 +21,6 @@ ij_kotlin_imports_layout = *, ^
 ij_kotlin_allow_trailing_comma = true
 ij_kotlin_allow_trailing_comma_on_call_site = true
 ktlint_code_style = android_studio
-ktlint_standard_function-naming = disabled
```

Comment:
It was disabled to lack of possibility to exclude `@Composable` function which starts uppercase